### PR TITLE
Add OTLP exporter configuration

### PR DESCRIPTION
**Pull request by nadiaciobanu**
Add an `options` struct to the OTLP exporter to allow exporter configuration (see [OTLP exporter spec](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/protocol/exporter.md)). Add a new OTLP exporter constructor that accepts and sets these options.

This PR adds an `endpoint` configuration, with a default of `localhost:55680`. In the future, other configuration options will be added, such as `timeout`.

Note that this PR changes the default port used to connect to the OpenTelemetry Collector from `55678` to `55680`, in accordance with https://github.com/open-telemetry/opentelemetry-cpp/issues/239.

### CODE REVIEW

File: exporters/otlp/include/opentelemetry/exporters/otlp/otlp_exporter.h [hunk 1]
```diff
@@ -8,18 +8,31 @@ namespace exporter
...
-   * Create an OtlpExporter. This constructor initializes a service stub to be
-   * used for exporting.
+   * Create an OtlpExporter using all default options.
    */
   OtlpExporter();
 
+  /**
+   * Create an OtlpExporter using the given options.
+   */
+  OtlpExporter(OtlpExporterOptions options);
```
Comment (reyang) [inline exporters/otlp/include/opentelemetry/exporters/otlp/otlp_exporter.h:34]: I think this probably should be a const pointer?

Comment (IlyaKobelevskiy) [reply exporters/otlp/include/opentelemetry/exporters/otlp/otlp_exporter.h:34]: +1 for const ref

File: exporters/otlp/test/otlp_exporter_test.cc [hunk 2]
```diff
@@ -23,6 +23,12 @@ class OtlpExporterTestPeer : public ::testing::Test
   {
     return std::unique_ptr<sdk::trace::SpanExporter>(new OtlpExporter(std::move(stub_interface)));
   }
+
+  // Get the options associated with the given exporter.
+  OtlpExporterOptions GetOptions(std::unique_ptr<OtlpExporter> &exporter)
```
Comment (IlyaKobelevskiy) [inline exporters/otlp/test/otlp_exporter_test.cc:28]: nit: This can be const OtlpExporterOptions&

**Comment by nadiaciobanu** [comment]
@reyang Ready to merge!

**Pull request merged**